boxoffice/views/order.py

```diff
@@ -480,7 +480,7 @@ def receipt(order: Order):
         LineItem.order == order, LineItem.status == LineItemStatus.CONFIRMED
     ).all()
     return render_template(
-        'cash_receipt.html.jinja2',
+        'cash_receipt_new.html.jinja2',
```

Comment:
Remove the old file if it's not used anymore. It's contents can be accessed from Git history.